csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "NuSpec.ReferenceGenerator": { "type": "build", "version": "2.0.0-beta-bld14" }
+  },
+  "frameworks": {
+    "netstandard1.0": {}
```

Comment:
I would be astonished if xbuild, monodevelop and xamarin studio didn't at least consume netstandard1.0 packages - if not now, then soon. I'd hope that they're using "vanilla" nuget somehow, and can therefore be updated by updating nuget, but I don't know for certain.

But it's xbuild that's failing to *build* this package right now.

Without these changes - or at least *some* changes - we don't get clean support of .NET Core. You can get it to work with imports, but that's ugly.

I really hope we don't end up having to decide between .NET Core and Xamarin... that would be an odd situation.